src/main/java/se/liu/ida/hefquin/engine/queryproc/impl/loptimizer/heuristics/formula/Formula.java

```diff
@@ -0,0 +1,11 @@
+package se.liu.ida.hefquin.engine.queryproc.impl.loptimizer.heuristics.formula;
+
+import se.liu.ida.hefquin.engine.queryproc.impl.loptimizer.heuristics.utils.QueryAnalyzer;
+
+import java.util.List;
+
+public interface Formula {
```

Comment:
Can you please give this interface a more specific name. "Formula" is very generic ;-)